x-pack/test/api_integration/config.js

```diff
@@ -26,9 +26,6 @@ export async function getApiIntegrationConfig({ readConfigFile }) {
         ...xPackFunctionalTestsConfig.get('kbnTestServer.serverArgs'),
         '--xpack.security.session.idleTimeout=3600000', // 1 hour
         '--optimize.enabled=false',
-        '--xpack.endpoint.enabled=true',
-        '--xpack.ingestManager.enabled=true',
-        '--xpack.ingestManager.fleet.enabled=true',
```

Comment:
These will need to be enabled for functional tests, and are included by the above `readfile` line in `api_integration` tests so they don't need to be duplicated.